pkg/plugin/plugin.go

```diff
@@ -205,17 +210,20 @@ func makeAutoscaleEnforcerPlugin(
 		return nil, fmt.Errorf("Error reading cluster state: %w", err)
 	}
 
-	go func() {
-		for {
-			callback, err := queue.Wait(ctx) // NB: Wait pulls from the front of the queue
-			if err != nil {
-				logger.Info("Stopped waiting on pod/VM queue", zap.Error(err))
-				break
-			}
+	for i := 0; i < config.EventQueueWorkers; i += 1 {
+		// copy the loop variable to avoid it escaping pre Go 1.22
+		go func(idx int) {
```

Comment:
nit: it's good to add ctx as a argument for this goroutine so it would be clear that we respect the context. At first, I thought we have no control over this goroutine's lifetime.

Reply:
fair enough, makes sense - done